초기 Deploy 설정 버전과의 충돌

### Status

- 초기 설정 버전들과의 충돌

   ```
   /home/ubuntu/.nvm/versions/node/v14.17.6/bin/node
   /usr/local/bin/node
   /usr/bin/node
   ```

### Todo List

- nvm 삭제 및 삭제 후 이슈 처리

기존 설정 관련 에러 발생
```
0|jenkins- | Error: Cannot find module '/home/ubuntu/.nvm/versions/node/v14.17.6/lib/node_modules/pm2/lib/ProcessContainerFork.js'
0|jenkins- |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
0|jenkins- |     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
0|jenkins- |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
0|jenkins- |     at node:internal/main/run_main_module:17:47 {
0|jenkins- |   code: 'MODULE_NOT_FOUND',
0|jenkins- |   requireStack: []
0|jenkins- | }
```

--- a/script/deploy
+++ b/script/deploy
@@ -1,5 +1,6 @@
 #!/bin/sh
 ssh ubuntu@3.34.124.94 <<EOF
+    sudo n lts
     cd ~/jenkins-test
     git pull origin master
     pm2 startOrRestart ecosystem.config.js

